pkg/micropython/doc.txt

```diff
@@ -0,0 +1,41 @@
+/**
+ * @defgroup pkg_micropython MicroPython RIOT port
+ * @ingroup  pkg
+ * @brief    MicroPython - Python for microcontrollers
+ *
+ * # MicroPython RIOT package
+ *
+ * "MicroPython is a lean and efficient implementation of the Python 3
+ *  programming language that includes a small subset of the Python standard
+ *  library and is optimised to run on microcontrollers and in constrained
+ *  environments."
+ *
+ * @see https://micropython.org
+ *
+ * ## Status
+ *
+ * MicroPython on RIOT has to be considered experimental. While the basic
+ * interpreter works fairly well on native an Cortex-M, it has not seen much
+ * testing. Especially GC and interrupt handling need some work.
+ *
+ * ## Available modules
+ *
+ * Currently, only these are implemented:
+ *
+ * ### thread_getpid()
+ *
+ *     $ import riot
+ *     $ print(riot.thread_getpid())
```

Comment:
Would it be possible to map this to the `os` module instead of having RIOT-specific names?

Reply:
Probably. There's no concept yet on how to map or integrate the (very pyboard specific) upstream API.